spec/layers.md

```diff
@@ -0,0 +1,81 @@
+clear function
+==============
+**Status**: explainer.
+
+Provide a `beginLayer()` and `endLayer()` functions that create and close layers for Canvas.
+
+
+Rationale
+---------
+
+Currently, the only way to draw a series of drawing calls at the same time, while applying the same sort of effects (shadows, blur, alpha, filters, compositing) to all of the drawn objects, without having these effects being applied one to one to this object, is by using an auxiliary canvas and then, drawing that auxiliary canvas to the canvas.
+This proposal adds an alternative to that (the same way that Fullter in Google and Core Graphics at Apple do) by creating a layer that then will be drawn in one go when the layer ends.
+
+
+Proposal
+--------
+
+```webidl
+interface CanvasRenderingContext2D {
+  // extending original
+  void beginLayer();
+  void endLayer();
+};
+
+interface OffscreenCanvasRenderingContext2D {
+  // extending original
+  void beginLayer();
+  void endLayer();
+};
+```
+
+`beginLayer()` sets the start point of the layer, it will also specify the effects (shadows, blur, alpha, filters, compositing) that will be used to draw the layer.
```

Comment:
maybe rephrase this by saying that the current state of the canvas specifies the effects that will be used. And list specifically  which part of the state is considered and which part is not.

Also, that we "reset" that part of the state inside the layer.